src/test/modelmanager_test.cpp

```diff
@@ -461,6 +461,17 @@ TEST(ModelManager, StartFromFile) {
     modelMock.reset();
 }
 
+TEST(ModelManager, StartFromFileWhenModelFilesMissing) {
+    std::filesystem::create_directories(model_1_path + "/1");
+    std::string fileToReload = "/tmp/ovms_config_file1.json";
```

Comment:
Maybe adding assertion of model version directory emptiness(filesystem::is_empty) would be a good idea since you are using same dir as in other tests.